db/re/mob_skill_db.txt

```diff
@@ -14164,6 +14164,165 @@
 20603,ABYSSMAN@NPC_COMBOATTACK,attack,171,2,2000,500,5000,no,target,always,0,,,,,,,
 20603,ABYSSMAN@NPC_BLOODDRAIN,attack,199,1,500,0,5000,yes,target,always,0,,,,,,29,
 
+// Illusion of Labyrinth
+20520,ILL_BAPHOMET@NPC_DARKSTRIKE,chase,340,10,2000,0,1000,yes,target,always,0,,,,,,,
+20520,ILL_BAPHOMET@AL_TELEPORT,idle,26,1,10000,0,0,yes,self,rudeattacked,0,,,,,,,
+20520,ILL_BAPHOMET@AL_TELEPORT,walk,26,1,5000,0,5000,yes,self,rudeattacked,0,,,,,,,
+20520,ILL_BAPHOMET@KN_BRANDISHSPEAR,attack,57,10,2000,500,5000,no,target,always,0,,,,,,29,
+20520,ILL_BAPHOMET@NPC_EARTHQUAKE,chase,653,5,10000,1500,30000,no,self,myhpltmaxrate,80,,,,,,6,
+20520,ILL_BAPHOMET@NPC_EARTHQUAKE,attack,653,5,10000,1500,30000,no,self,myhpltmaxrate,80,,,,,,6,
+// 20520,ILL_BAPHOMET@NPC_EARTHQUAKE_K,chase,750,5,10000,1500,30000,no,self,myhpltmaxrate,80,,,,,,6,
+// 20520,ILL_BAPHOMET@NPC_EARTHQUAKE_K,attack,750,5,10000,1500,30000,no,self,myhpltmaxrate,80,,,,,,6,
+20520,ILL_BAPHOMET@NPC_POWERUP,attack,349,5,10000,0,30000,yes,self,myhpltmaxrate,30,,,,,,6,
+20520,ILL_BAPHOMET@NPC_CALLSLAVE,attack,352,1,10000,0,30000,yes,self,always,0,,,,,,,
+20520,ILL_BAPHOMET@NPC_CALLSLAVE,idle,352,1,10000,0,30000,yes,self,always,0,,,,,,,
+20520,ILL_BAPHOMET@NPC_HELLJUDGEMENT2,chase,768,5,10000,800,10000,no,target,always,0,,,,,,6,
+20520,ILL_BAPHOMET@NPC_HELLJUDGEMENT2,attack,768,5,10000,800,10000,no,target,always,0,,,,,,6,
+20520,ILL_BAPHOMET@NPC_DARKBREATH,attack,202,5,2000,800,5000,no,target,always,0,,,,,,29,
+20520,ILL_BAPHOMET@NPC_ARMORBRAKE,attack,344,10,2000,0,5000,no,target,always,0,,,,,,,
+20520,ILL_BAPHOMET@NPC_GUIDEDATTACK,attack,172,5,500,0,20000,no,target,always,0,,,,,,,
+20520,ILL_BAPHOMET@NPC_SUMMONSLAVE,attack,196,1,10000,700,10000,no,self,slavele,3,20533,,,,,,
+20520,ILL_BAPHOMET@NPC_SUMMONSLAVE,idle,196,1,10000,700,10000,no,self,slavele,3,20533,,,,,,
+20520,ILL_BAPHOMET@WZ_VERMILION,attack,85,21,2000,500,2000,no,target,always,0,,,,,,29,
+20520,ILL_BAPHOMET@WZ_VERMILION,chase,85,21,5000,500,2000,no,target,skillused,18,,,,,,29,
+20520,ILL_BAPHOMET@WZ_VERMILION,chase,85,21,2000,500,2000,no,target,always,0,,,,,,29,
+20520,ILL_BAPHOMET@AL_HEAL,idle,28,11,10000,0,5000,yes,friend,myhpltmaxrate,50,,,,,,,
+20520,ILL_BAPHOMET@NPC_CRITICALWOUND,attack,673,3,500,0,5000,yes,target,always,0,,,,,,,
+20520,ILL_BAPHOMET@NPC_BLEEDING2,attack,764,3,500,0,5000,yes,target,always,0,,,,,,,
+20520,ILL_BAPHOMET@NPC_GRADUAL_GRAVITY,attack,752,3,3000,0,5000,yes,target,myhpltmaxrate,20,,,,,,,
+20520,ILL_BAPHOMET@NPC_DAMAGE_HEAL,attack,753,1,10000,1500,60000,no,target,myhpltmaxrate,10,,,,,,,
+20521,ILL_ANDREA@AL_PNEUMA,idle,25,1,5000,0,10000,yes,target,longrangeattacked,0,,,,,,22,
```

Comment:
Using AL_PNEUMA as a target seems to be incorrect!

Reply:
Correct, it's incorrect!